flink-cdc-common/src/main/java/com/ververica/cdc/common/event/AddColumnEvent.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.common.event;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+import com.ververica.cdc.common.schema.Column;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * A {@link SchemaChangeEvent} that represents an {@code ADD COLUMN} DDL, which may contain the
+ * lenient column type changes.
+ */
+@PublicEvolving
+public final class AddColumnEvent implements SchemaChangeEvent, Serializable {
```

Comment:
`SchemaChangeEvent` already extends `Serializable`. We do not need to implement `Serializable` here.